backend/db/conn.js

```diff
@@ -1,29 +1,37 @@
 const { MongoClient } = require("mongodb");
 const uri = process.env.MONGODB_URI;
-const client = new MongoClient(uri, {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-});
+const client1 = new MongoClient(uri);
+const client2 = new MongoClient(uri);
 
-let _db;
+let _db1;
+let _db2;
 
 module.exports = {
   connectToServer: async function (callback) {
-    try{ 
-      await client.connect();
+    try {
+      await client1.connect();
+      await client2.connect();
     } catch (e) {
       console.error(e);
     }
 
-    _db = client.db("myDB")
+    _db1 = client1.db("logging");
```

Comment:
Changes made here will affect the connection with the existing two connections for login and signup

Reply:
yes, will need to change the connection to fit the database design schema